### DISCUSSION TITLE
nni2.2 error - UnicodeDecodeError

### DISCUSSION THREAD

**Discussion by lym-zyl** [microsoft/nni#3804] [Q&A] [answered]
**Describe the issue**:

**Environment**:
- NNI version:2.2
- Training service (local|remote|pai|aml|etc):local
- Client OS:
- Server OS (for remote mode only):
- Python version:3.8.8
- Is conda/virtualenv/venv used?: anaconda3, jupyter
- Is running in Docker?:no

**Configuration**:
 - Experiment config (remember to remove secrets!):nni"s trail "sklearn"
 - Search space:nni"s trail "sklearn"

**Log message**:
![image](https://user-images.githubusercontent.com/81909301/117150257-8f1e1d00-adea-11eb-8d89-5e8b5dc61563.png)

 

**How to reproduce it?**:

**Comment by SparkSnail** [comment, marked as answer]
@lym-zyl this error caused by invalid json format of your nni experiment meta. could you please check the if content of "C:\users\nni-experiment\.experiment" file is valid json? You could also delete this file and restart NNI experiment.
